lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/cc/PrintHandler.java

```diff
@@ -569,6 +576,28 @@ public void setCCItemXml(Element itemXml, Element resourceXml, AbstractParser pa
       boolean inline = false;
       String mmDisplayType = null;
       String resourceId = resourceXml.getAttributeValue(IDENTIFIER);
+      log.debug("identifier {} identifierRef {} resourceId {} resourceXml {}", identifier, identifierRef, resourceId, resourceXml);
+
+      // Check if there is any Canvas module metadata in the import
+      Element canvas_module_item = null;
+      boolean open_same_window = true;
+      if ( identifier != null && canvasModuleMeta != null ) {
+
+          Namespace namespace = Namespace.getNamespace("cccv1p0", CANVAS_MODULE_META_NAMESPACE);
+
+          String xpathExpression = "//cccv1p0:item[@identifier='" + identifier + "']";
+
+          List<Element> items = XPathFactory.instance().compile(xpathExpression, Filters.element(), null, namespace).evaluate(canvasModuleMeta);
+
+          log.debug("Found {} items whilst looking for {}", items.size(), xpathExpression);
+          if ( items.size() >= 1 ) {
+              canvas_module_item = items.get(0);
+              String new_tab = canvas_module_item.getChildText("new_tab", namespace);
+              open_same_window = "false".equals(new_tab);
```

Comment:
Boolean.FALSE.toString()

Reply:
This is not a Java false.   It is part of the Canvas spec - it is a string.  I will make the strings constants to clarify that these are not Java constants but Canvas constants.